pkg/crcontrollers/operator/Dockerfile

```diff
@@ -2,6 +2,18 @@ FROM    ubuntu:20.04
 
 LABEL   description="Baremetal CSI Operator"
 
+ENV BASE_URL="https://get.helm.sh"
+ENV TAR_FILE="helm-v3.0.0-linux-amd64.tar.gz"
+
+RUN apt update --no-install-recommends -y -q && \
+    apt install --no-install-recommends -y -q curl ca-certificates && \
+    curl -L ${BASE_URL}/${TAR_FILE} |tar xvz && \
+    mv linux-amd64/helm /usr/bin/helm && \
```

Comment:
Please add a comment to revert these changes later.

Reply:
Done